webpack/webpack-deep-imports-compat-plugin/src/DeepImportsCompatPlugin.ts

```diff
@@ -1,64 +1,47 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import type {
-  WebpackPluginInstance,
-  Compiler,
-  Configuration,
-  Chunk,
-  WebpackError as WebpackErrorType
-} from 'webpack';
-import VirtualModulesPlugin from 'webpack-virtual-modules';
+import { DllPlugin, type Compiler, WebpackError, Chunk, NormalModule } from 'webpack';
 import path from 'path';
-import glob from 'fast-glob';
-import { Path } from '@rushstack/node-core-library';
+import { Async, FileSystem, LegacyAdapters, Path } from '@rushstack/node-core-library';
 
 const PLUGIN_NAME: 'DeepImportsCompatPlugin' = 'DeepImportsCompatPlugin';
 
+declare const dummyDllPlugin: DllPlugin;
+type DllPluginOptions = typeof dummyDllPlugin.options;
```

Comment:
Why do we need `dummyDllPlugin`?

Reply:
The type of the constructor options for `DllPlugin` aren't exported from Webpack's typings. By declaring a fake instance of `DllPlugin`, we can get at the options type by referencing the type of the instance's `options` property.